pkg/backend/kubeflow/generator/cm.go

```diff
@@ -18,8 +18,9 @@ import (
 	"fmt"
 
 	s2iconfigmap "github.com/caicloud/ciao/pkg/s2i/configmap"
-	pytorchv1alpha2 "github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1alpha2"
-	tfv1alpha2 "github.com/kubeflow/tf-operator/pkg/apis/tensorflow/v1alpha2"
+	common "github.com/kubeflow/tf-operator/pkg/apis/common/v1beta2"
+	pytorchv1beta2 "github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1beta2"
```

Comment:
pytorch and tf does not share the same common now, then should we import two different common package?